docs/map.xsd

```diff
@@ -247,22 +452,33 @@
 </xs:attributeGroup>
 
 <xs:attributeGroup name="tile.data.layer">
-  <xs:attribute name="gid" type="xs:nonNegativeInteger" use="required"/>
+  <xs:attribute name="gid" type="tileId" use="required"/>
 </xs:attributeGroup>
 
 <xs:attributeGroup name="objectgroup">
+  <!--Not sure if missing x,y which are defaulted to 0 and unchangable-->
   <xs:attribute name="name" type="nameT" use="required"/>
-  <xs:attribute name="width" type="xs:nonNegativeInteger" use="required"/>
-  <xs:attribute name="height" type="xs:nonNegativeInteger" use="required"/>
+  <xs:attribute name="width" type="pixelT" use="required"/>
+  <xs:attribute name="height" type="pixelT" use="required"/>
```

Comment:
`objectgroup` does not require `width` and `height`. In fact these properties are deprecated, not written out and ignored by Tiled. I guess we should just remove them here?